### PR TITLE
add special rewarding/staking pool address

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: address/address_v1_test.go [hunk 1]
```diff
@@ -65,6 +65,35 @@ func TestAddressError(t *testing.T) {
 	encodedAddrBytes := []byte(encodedAddr)
 	encodedAddrBytes[len(encodedAddrBytes)-1] = 'o'
 	addr2, err := _v1.FromString(string(encodedAddrBytes))
-	assert.Nil(t, addr2)
-	assert.Error(t, err)
+	require.Nil(t, addr2)
+	require.Error(t, err)
+}
+
+func TestEtherCompatibility(t *testing.T) {
```
Comment (dustinxie) [inline address/address_v1_test.go:72]: an earlier test that should check-in

File: address/address_v1.go [hunk 2]
```diff
@@ -15,11 +15,23 @@ import (
 	"github.com/iotexproject/iotex-address/address/bech32"
 )
 
+// special rewarding/staking pool address
+const (
+	StakingBucketPoolAddr = "io000000000000000000000000stakingprotocol"
```
Comment (yutongp) [inline address/address_v1.go:20]: Should these grouped with zeroAddr?

File: address/address_v1_test.go [hunk 3]
```diff
@@ -11,8 +11,8 @@ import (
 	"strings"
 	"testing"
 
+	ether "github.com/ethereum/go-ethereum/crypto"
```
Comment (yutongp) [inline address/address_v1_test.go:14]: ether> ethcrypto

File: address/address_v1.go (hunk 2 above)
Comment (dustinxie) [reply address/address_v1.go:20]: fixed

File: address/address_v1_test.go (hunk 3 above)
Comment (dustinxie) [reply address/address_v1_test.go:14]: fixed

**Pull request merged**